tests/components/shelly/test_binary_sensor.py

```diff
@@ -143,18 +159,23 @@ async def test_block_sleeping_binary_sensor(
     await init_integration(hass, 1, sleep_period=1000)
 
     # Sensor should be created when device is online
-    assert hass.states.get(entity_id) is None
+    state = hass.states.get(entity_id)
+    assert state is None
```

Comment:
I don't understand the reason to split this into two lines
```suggestion
    assert hass.states.get(entity_id) is None
```

Reply:
Was just to align to other places and to the same syntax used for entry.
Will update all places of the PR to the one-line syntax